### DISCUSSION TITLE
Maya segfaults on both Linux (WSL) and Mac

### DISCUSSION THREAD

**Discussion by drewbrew** [djangocon/2021.djangocon.us#55] [Q&A]
```bash
% pip install maya
% python -c 'import maya; print(maya.__version__)'
0.6.1
% python --version
Python 3.9.6
% python -c 'import maya; print(maya.when("2021-10-22 10:00"))'
zsh: segmentation fault  python -c 'import maya; print(maya.when("2021-10-22 10:00"))'
```

If I change the offending line in [process.py](https://github.com/djangocon/2021.djangocon.us/commit/050f1fc410cff1b8b57e3200a76bb4ffdce03284#diff-fa6a38a42e34916c7bda26872f4d700584391dbb9f5fb958b273134251dc745aR21-R23) to:

```python
from dateutil.parser import parse
date = parse(post["date"])
```

Then the script runs but I get off by 12 hours, which is better than nothing.

**Comment by jefftriplett** [comment]
@drewbrew are you on mac or windows? 

```shell
❯ python --version
Python 3.9.7
2021.djangocon.us-git on  drop-note-about-tz [$!?] via 🐍 v3.9.7 (djangocon) via 💎 v2.7.2 on ☁️  (us-west-2) on ☁️  jeff@revsys.com
❯ python -c 'import maya; print(maya.when("2021-10-22 10:00"))'
Fri, 22 Oct 2021 10:00:00 GMT
```

**Reply by jefftriplett** [reply]
If it gets in your way, I'm +1 for using something better. 

That said, I'm -1 for changing anything if it renames our filenames or shifts times slightly since that's our focus in the short window we have left.

**Reply by drewbrew** [reply]
I'm on macOS Big Sur. That being said, this is not worth caring about for the rest of the year. We're good for now and we can worry about that next year.
